react-native-ssi/src/types.ts

```diff
@@ -96,3 +96,187 @@ export type TermsOfUse = {
 export type Proof = {
   type: string | string[];
 };
+
+export type CredentialManifest = {
+  id: string;
+  spec_version: string;
+  issuer: CredentialIssuer;
+  output_descriptors: OutputDescriptor[];
+  name?: string;
+  description?: string;
+  format?: ClaimFormatDesignation;
+  presentation_definition?: PresentationDefinition;
+};
+
+export type CredentialIssuer = {
+  id: string;
+  name?: string;
+  styles?: EntityStyle;
+  comment?: string;
+};
+
+export type EntityStyleImage = {
+  uri: string;
+  alt?: string;
+};
+
+export type EntityStyleColor = {
+  color: string;
+};
+
+export type EntityStyle = {
+  thumbnail?: EntityStyleImage;
+  hero?: EntityStyleImage;
+  background?: EntityStyleColor;
+  text?: EntityStyleColor;
+};
+
+export type OutputDescriptor = {
+  id: string;
+  schema: string;
+  name?: string;
+  description?: string;
+  styles?: EntityStyle;
```

Comment:
`styles?: OutputDescriptorStyle`

Reply:
Ahh, I see what you meant by the last comment now. The style is also defined on `CredentialIssuer`, which is why verite had it defined as `EntityStyle`.